vcpkg.json

```diff
@@ -43,10 +46,16 @@
         "opencl": {
             "description": "Build OpenCL backend",
             "dependencies": [
-                "boost-program-options",
+                "boost-compute",
                 "opencl"
             ]
+        },
+        "mkl": {
+            "description": "Build with MKL",
+            "dependencies": [
+                "intel-mkl"
+            ]
```

Comment:
have you tested these combinations of theses features ? some times they may not work as one intuitively intends them to with vcpkg (at least from my past experience)

Reply:
The features are enabled when you set the AF_BUILD_* variables in cmake. I haven't seen an issue with it locall. The only problem I had was with cuDNN but I made a PR to fix that in the vcpkg repo.